Gemfile

```
@@ -20,7 +20,9 @@ gem 'scoped_search', '>= 4.1.8', '< 5'
 gem 'ldap_fluff', '>= 0.5.0', '< 1.0'
 gem 'apipie-rails', '>= 0.5.17', '< 0.6.0'
 gem 'apipie-dsl', '>= 2.2.6'
-gem 'rdoc'
+# Pin rdoc to prevent psych bundle (https://github.com/ruby/rdoc/commit/ebe185c8775b2afe844eb3da6fa78adaa79e29a4)
+# Pin psych is needed for Rails 6.0, Rails 6.1 should work with Psych > 4
+gem 'rdoc', '~> 6.3.3'
```

I'd prefer:
```suggestion
gem 'rdoc', '< 6.4'
```

This is closer to what we actually want. However, it's really a slight nit.